### PR TITLE
add 10k German News Article Dataset

### PR DESCRIPTION
**Pull request by stevhliu**

### CODE REVIEW

File: datasets/gnad_10/gnad_10.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+_HOMEPAGE = "https://tblock.github.io/10kGNAD/"
+
+_LICENSE = "Creative Commons Attribution-NonCommercial-ShareAlike 4.0"
+
+_TRAIN_DOWNLOAD_URL = "https://raw.githubusercontent.com/tblock/10kGNAD/master/train.csv"
+_TEST_DOWNLOAD_URL = "https://raw.githubusercontent.com/tblock/10kGNAD/master/test.csv"
+
+
+class GNAD_10(datasets.GeneratorBasedBuilder):
```
Comment (lhoestq) [inline datasets/gnad_10/gnad_10.py:41]:
```suggestion
class Gnad10(datasets.GeneratorBasedBuilder):
```
can you rename this class please ?

can you also rename the dataset folder and the dataset script to `gnad10` ?

File: datasets/kor_hate/README.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
+---
```
Comment (lhoestq) [inline datasets/kor_hate/README.md:1]: could you only include files related to gnad 10 in this PR please ?

**Review by lhoestq — COMMENTED** [review]
Really cool ! thanks

I just left one comment about the names of the class and the script/folder
Can you do this change please ?
You'll need to regenerate the dataset_infos.json after this change

Moreover it looks like this PR also includes changes about other datasets.
Could you only include changes about gnad 10 in this PR please ?

File: datasets/gnad_10/gnad_10.py (hunk 1 above)
Comment (stevhliu) [reply datasets/gnad_10/gnad_10.py]: thanks @lhoestq! i've renamed everything to reflect `gnad10`, but i can't figure out how to include changes that are only related to this PR (i'm going to be on the road for a few days but i will be back monday).

**Comment by lhoestq** [comment]
You can just create another branch from master on your fork and create another PR:

first update your master branch
```
git checkout master
git fetch upstream
git rebase upstream/master
git push
```

then create a new branch
```
git checkout -b my-new-branch-name
```

Then you can add, commit and push the gnad10 files and open a new PR

**Comment by lhoestq** [comment]
closing in favor of #1572

**Pull request closed without merging**